lib/t/calls.ex

```diff
@@ -213,6 +213,10 @@ defmodule T.Calls do
         seconds = call.ended_at |> DateTime.diff(call.accepted_at)
         minutes = div(seconds, 60)
 
+        if minutes > 5 do
+          T.Matches.match_after_live_call([call.caller_id, call.called_id])
```

Comment:
@akhrail1996 is this enough to decide if the call has been successful?

Reply:
I guess even 1 minute call is enough